src/app/pages/dashboard/widgets/system/common/product-image/product-image.component.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  Component, ChangeDetectionStrategy, input,
+  computed,
+  HostBinding,
+} from '@angular/core';
+import { Router } from '@angular/router';
+import { getProductEnclosure, getProductImage, getServerProduct } from 'app/pages/dashboard/widgets/system/common/widget-sys-info.utils';
+
+@Component({
+  selector: 'ix-product-image',
+  templateUrl: './product-image.component.html',
+  styleUrls: ['./product-image.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ProductImageComponent {
+  @HostBinding('class.ix-logo') get isIxLogo(): boolean {
+    return this.productImage().includes('ix-original.svg');
+  }
+  @HostBinding('class.truenas')
+  isEnterprise = input.required<boolean>();
+  systemProduct = input.required<string>();
+  hasEnclosureSupport = input.required<boolean>();
+  isHaLicensed = input.required<boolean>();
+  isIxHardware = input.required<boolean>();
+
+  isUnsupportedHardware = computed(() => this.isEnterprise() && !this.isIxHardware() && !this.isHaLicensed());
```

Comment:
Prefer writing this like the following
```
isUnsupportedHardware = computed(() => {
  const isEnterprise = this.isEnterprise();
  const isIxHardware = this.isIxHardware();
  const isHaLicensed = this.isHaLicensed();
  return isEnterprise && !isIxHardware && !isHaLicensed;
});
```
Because of the way JavaScript handles comparison operators, it can ignore a signal dependency in a computed signal if during the initial execution, the signal was never fetched. For example, `computed(() => this.isEnterprise() || this.isHa())` will ignore `this.isHa()` every time if `this.isEnterprise()` is always true. So, `this.isHa()` is never considered a dependency for the computed signal.
This applies to all computed signals. As a best practice, any dependency signals should be fetched at the start of the compute callback function into variables so it's clear that they are all dependencies.